src/Integrations/Integrations/Symfony/SymfonyIntegration.php

```diff
@@ -357,23 +349,28 @@ function (SpanData $span, $args, $response) use ($integration) {
                 $span->type = Type::WEB_SERVLET;
                 $span->meta[Tag::COMPONENT] = SymfonyIntegration::NAME;
 
-                $integration->symfonyRequestSpan->meta[Tag::HTTP_METHOD] = $request->getMethod();
-                $integration->symfonyRequestSpan->meta[Tag::COMPONENT] = SymfonyIntegration::NAME;
-                $integration->symfonyRequestSpan->meta[Tag::SPAN_KIND] = 'server';
+                $rootSpan = root_span();
+                if ($rootSpan === null) {
+                    return;
+                }
```

Comment:
```suggestion
```
Redundant: in trace_method

Reply:
This check happens in a hook on `Symfony\Component\HttpKernel\HttpKernel::handle`, which is the same method as the one on line 40 of the same file. I'm not totally convinced that this check should be removed, to be 100% honest.